packages/renderer/index.d.ts

```diff
@@ -236,7 +236,7 @@ declare namespace ReactPDF {
 
   class Canvas extends React.Component<CanvasProps> {}
 
-  interface SVGProps extends NodeProps {
+  interface SVGProps extends NodeProps, SVGPresentationAttributes {
```

Comment:
I'm not sure SVG supports all the props in `SVGPresentationAttributes`. All other fixes look correct

Reply:
Correct me if I'm wrong, but I just checked the [w3 docs](https://www.w3.org/TR/SVG/styling.html#PresentationAttributes), and it seems like SVG supports all Presentation Attributes, except some Geometry Properties like `cx`, `cy`, `r`, `rx`, and `d`. Also, if you try using the native `<svg>` element, the TS will autocomplete all the props in `SVGPresentationAttributes`.

Reply:
I'm not 100% sure our `<Svg />` component supports passing all of these, but I'll include it. I don't actively maintain types tbh, so I'll let base it on what the community finds :)